package.json

```diff
@@ -73,6 +73,7 @@
     "mocha": "^7.1.1",
     "npm-run-all": "^4.1.5",
     "playwright": "^1.2.1",
+    "playwright-chromium": "^1.8.0-next.1608058261773",
```

Comment:
why is that needed? 🤔

Reply:
Hmm we should remove that again, it's only needed if you're on mac big sur